backend/controllers/attachements_controller.py

```diff
@@ -50,3 +50,36 @@ def upload_files(dev_id: int) -> tuple[Response, int]:
 
     return (jsonify({"message": "Files uploaded successfully", "files": saved_files}),
             200)
+
+
+def get_all_files_in_directory(directory_path: str) -> list[str]:
+    try:
+        # List all files and directories in the specified path
+        all_items = os.listdir(directory_path)
+        # Filter out only the files
+        files = [
+            item for item in all_items
+            if os.path.isfile(os.path.join(directory_path, item))
+        ]
+        return files
+    except OSError as e:
+        print(f"Error accessing directory: {e}")
+        return []  # Return an empty list in case of an error
+
+
+def list_files(dev_id: int) -> tuple[Response, int]:
+    device_attachment_directory = os.path.join('static', 'attachments', str(dev_id))
+
+    all_files = get_all_files_in_directory(device_attachment_directory)
+
+    if not os.path.exists(device_attachment_directory):
+        return jsonify({"error": "Directory not found"}), 404
```

Comment:
Nää taitaa olla väärinpäin? Checkkaa ekana että onko pathia olemassa ja sitte vasta listaa filejä.